src/ts/component/editor/page.tsx

```diff
@@ -793,10 +793,18 @@ const EditorPage = observer(class EditorPage extends React.Component<Props, Stat
 			});
 
 			if (block.isTextToggle()) {
-				keyboard.shortcut(`${cmd}+shift+t`, e, () => {
+				keyboard.shortcut(`${cmd}+enter`, e, () => {
 					blockStore.toggle(rootId, block.id, !Storage.checkToggle(rootId, block.id));
 				});
 			};
+
+			if (block.isTextCheckbox()) {
+				keyboard.shortcut(`${cmd}+enter`, e, () => {
+					UtilData.blockSetText(rootId, block.id, text, marks, true, () => {
+						C.BlockTextSetChecked(rootId, block.id, !block.content.checked);
+					});
+				});
+			};
```

Comment:
This code should be combined with the code above, its only different in Api command

Reply:
I've pushed new code separating the shortcuts.